onset_maproom/environment_linux.yml

```diff
@@ -1,4 +1,4 @@
-name: enactsmaproom
+name: onset
```

Comment:
I would like to continue to call this enactsmaproom. There are eventually dozens of ENACTS maprooms, and I don't want them each to have their own conda environment.